inst/rmarkdown/templates/xaringan/resources/rutgers-fonts.css

```diff
@@ -0,0 +1,15 @@
+
+/* Add maven for code font (keep other xaringan defaults */
+@import url(https://fonts.googleapis.com/css?family=Maven+Pro);
```

Comment:
This font does not seem to be used anywhere.

Reply:
It looks like I can just remove the `rutgers-fonts.css` file.